client/js/bundles/statement/assessmentTableView.js

```diff
@@ -34,4 +33,8 @@ const components = {
   DpTable
 }
 
+if (hasPermission('area_admin_boilerplates')) {
```

Comment:
Do we need to import `hasPermission` here?
- If yes, the import is missing in the files below, too. 
- If not, can we remove the import from the files above?

Reply:
fixed in f38170fcbc35bb136a7f6770a32d31c05451a7d2

Reply:
It only has to be imported in entrypoints. In the vue-context its already available. I hope now I added all missing ones-